requirements.txt

```diff
@@ -2,6 +2,6 @@ matplotlib==3.3.4
 numpy==1.19.5
 pandas==1.1.5
 scikit-learn==0.24.2
-scipy==1.5.4
+scipy
```

Comment:
I'm curious why you dropped the version requirement here.

Reply:
For me, natively, it failed with the version constraint, let me test inside the container.

Reply:
It's not needed, I reverted this out-of-scope change.

Reply:
Okay, great. Happy to update the version if there's a particular conflict or any security concern